### PR TITLE
Update Handwriting Recognition Article for Shipping

### PR DESCRIPTION
**Pull request by jpmedley**
Fixes #SOME_ISSUE_NUMBER

Changes proposed in this pull request:

-
-
-

When you're ready to submit your PR, don't forget to add the `$-presubmit` label.

### CODE REVIEW

File: src/site/content/en/blog/handwriting-recognition/index.md [hunk 1]
```diff
@@ -7,18 +7,17 @@ authors:
...
   - blog # blog is a required tag for the article to show up in the blog.
   - capabilities
-origin_trial:
-  url: https://developer.chrome.com/origintrials/#/view_trial/3207688834594635777
 ---
 
-{% Aside %} The Handwriting Recognition API is part of the [capabilities project](/fugu-status/) and
-is currently in development. This post will be updated as the implementation progresses.
+{% Aside %}
+The Handwriting Recognition API is part of the [capabilities project](/fugu-status/) and launched in Chrome&nbsp;99.
```
Comment (tomayac) [inline src/site/content/en/blog/handwriting-recognition/index.md:20]:
```suggestion
The Handwriting Recognition API is part of the [capabilities project](/fugu-status/) and launched in Chromium&nbsp;99.
```

File: src/site/content/en/blog/handwriting-recognition/index.md [hunk 2]
```diff
@@ -54,33 +53,13 @@ Example uses include:
 | 1. Create explainer                      | [Complete][explainer]    |
 | 2. Create initial draft of specification | [Complete][spec]         |
 | 3. Gather feedback & iterate on design   | [In progress](#feedback) |
-| 4. Origin trial                          | [In progress][ot]        |
-| 5. Launch                                | Not started              |
+| 4. Origin trial                          | Complete                 |
+| 5. Launch                                | Chrome 99                |
```
Comment (tomayac) [inline src/site/content/en/blog/handwriting-recognition/index.md:57]:
```suggestion
| 5. **Launch**                                | ** Complete** (Chromium&nbsp;99) |
```

**Review by tomayac — APPROVED** [review]
Just some minor nits, see the suggestions.

**Comment by jpmedley** [comment]
@tomayac PTAL

**Review by tomayac — APPROVED** [review]
LGTM

Run it through prettier to get the indentation right, it's messed up in the table. You can use [this link](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEIA0IIAcYEtoDOyoAhgE5kQDuACuQkSiQDZUkCeRGARmSWAGs4MAMokAtnAAyuKHGQAzFgTg8+g4SKz9ZAc2QwyAV1Ug447nAAmV61JJRdRkrrgAxCGXEkYeR8hASIxgIdBAACxhxZgB1cNx4Am0wOBEGBNwANwT2ALACLhBZFTIYGj5db0VlUwArAgAPET1mOABFIwh4auYVDG0yEoDvMgEraigwrDJZGBjcKxhw5ABGAAY1-soVGL4sAJY2TjCAR074cuxGQIIAWjlrazCyODPcF-KXKqQlXtMVcS4AzGf4tdrneRIQwmDAwEjceaLZZIABMsL4uGYegAwhBxN9AsxmGEjCoACrwxi-PogTImACSUFssBEYBmOAAgkyRDB2K0eioAL6CoA) to copy & paste it in.

**Review by samdutton — COMMENTED** [review]
LGTM.

**Pull request merged**